src/databricks/labs/ucx/runtime.py

```diff
@@ -187,15 +259,32 @@ def crawl_permissions(cfg: WorkspaceConfig):
 @task(
     "assessment",
     depends_on=[
-        crawl_grants,
-        crawl_permissions,
-        guess_external_locations,
         assess_jobs,
         assess_clusters,
         assess_pipelines,
         assess_azure_service_principals,
         assess_global_init_scripts,
+        crawl_tables,
     ],
+)
+def setup_view(cfg: WorkspaceConfig):
+    """Creates a database view for capturing the following failures as part of the assessment process:
+    - Unsupported DBR version
+    - Unsupported config
+    - DBFS mount used in configuration
+    - Azure service principal credentials used in config
+    - Unsupported storage type (WASBS, ADL) used in table location
+    - Table scan failure
+    """
+    backend = RuntimeBackend()
+    hms_db = cfg.replace_inventory_variable(cfg.inventory_database)
```

Comment:
Aren't you supposed to pass view definition to this method and retrieve query with replaced variable?